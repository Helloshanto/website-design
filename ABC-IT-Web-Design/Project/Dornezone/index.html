<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>DorneZone</title>

    <!-- font awesome -->
    <script src="https://kit.fontawesome.com/40aed592e2.js" crossorigin="anonymous"></script>
    <!--  -->

    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.7.2/animate.min.css">
    <!-- style -->
    <link rel="stylesheet" href="css/Custom.css">
    <link rel="stylesheet" href="css/style.css">
</head>

<body>
    <header>
        <div class="container">
            <div class="nav">
                <div class="logo fl animated bounceInLeft ">
                    <img src="img/logo.png" alt="">
                </div>
                <div class="menu fr animated bounceInRight ">
                    <ul>
                        <li><a href="#">Home</a></li>
                        <li><a href="#">About</a></li>
                        <li><a href="#">Pricing</a></li>
                        <li><a href="#">Our Dronse</a></li>
                        <li><a href="#">Our Realization</a></li>
                        <li><a href="#" class="menu-btn">Contact</a></li>
                    </ul>
                </div>
            </div>
            <div class="hero-area">
                <div class="hero-text animated bounceIn">
                    <h1>Film your event with us! </h1>
                    <p>Pellentesque eget nunc sit amet urna ullamcorper fermentum et eu leo.
                        Nunc vel nibh tempor, pharetra lectus congue, luctus orci.</p>
                    <div class="hero-btn">
                        <a href="#" class="hero-btn-active">Ask for price</a>
                        <a href="#"><i class="fas fa-play"></i> Watch video</a>
                    </div>
                </div>
            </div>
            <div class="mouse animated infinite fadeInDown  ">
                <a href="#doren-info">
                    <i class="fas fa-mouse"></i>
                </a>
                <p>scrool down</p>
            </div>
        </div>
    </header>
    <!-- Doren Information -->
    <section class="doren-info pd100" id="doren-info">
        <div class="container">
            <div class="col-25">
                <div class="doren-info-part ">
                    <img src="img/icon-1.png" alt="">
                    <h2>From air</h2>
                    <p>Pellentesque eget nunc sit amet urna ullamcorper fermentum et eu leo.</p>
                </div>
            </div>
            <div class="col-25">
                <div class="doren-info-part ">
                    <img src="img/icon-2.png" alt="">
                    <h2>From air</h2>
                    <p>Pellentesque eget nunc sit amet urna ullamcorper fermentum et eu leo.</p>
                </div>
            </div>
            <div class="col-25">
                <div class="doren-info-part ">
                    <img src="img/icon-3.png" alt="">
                    <h2>From air</h2>
                    <p>Pellentesque eget nunc sit amet urna ullamcorper fermentum et eu leo.</p>
                </div>
            </div>
            <div class="col-25">
                <div class="doren-info-part ">
                    <img src="img/icon-4.png" alt="">
                    <h2>From air</h2>
                    <p>Pellentesque eget nunc sit amet urna ullamcorper fermentum et eu leo.</p>
                </div>
            </div>
            <div class="btn pd50">
                <a href="#">Ask Me price</a>
            </div>
        </div>
    </section>
    <!-- END Doren Information -->
    <!--  Nature section -->
    <section class="nature pd100">
        <div class="container">
            <div class="col-70">
                <div class="section-head pd50">
                    <h1>Nature from the air</h1>
                    <p>Mauris consequat libero metus, nec ultricies sem efficitur quis. Integer bibendum eget metus ac.
                    </p>
                </div>

                <div class="nature-count">
                    <div class="col-40">
                        <div class="ncount-part ">
                            <h4 class="count-number">1</h4>
                            <p>Pellentesque eget nunc sit amet urna ullamcorper fermentum et eu leo. Nunc vel nibh</p>
                        </div>
                        <div class="ncount-part">
                            <h4 class="count-number">2</h4>
                            <p>Pellentesque eget nunc sit amet urna ullamcorper fermentum et eu leo. Nunc vel nibh</p>
                        </div>
                        <div class="ncount-part ">
                            <h4 class="count-number">3</h4>
                            <p>Pellentesque eget nunc sit amet urna ullamcorper fermentum et eu leo. Nunc vel nibh.</p>
                        </div>
                    </div>
                    <div class="col-30">
                        <div class="ncount-part ">
                            <h4 class="count-number">4</h4>
                            <p>Pellentesque eget nunc sit amet urna ullamcorper fermentum et eu</p>
                        </div>
                        <div class="ncount-part ">
                            <h4 class="count-number">5</h4>
                            <p>Pellentesque eget nunc sit amet urna ullamcorper fermentum et eu</p>
                        </div>
                    </div>
                </div>
            </div>
            <div class="doren-img col-30">
                <img src="img/Drone.png" alt="">
            </div>
        </div>
    </section>
    <!--END Nature Section-->
    <!-- dorenvideo -->
    <section class="dorenvideo ">
        <div class="container">
            <div class="video-top">
                <div class="video-section fl ">
                    <img src="img/img-left.png" alt="">
                </div>
                <div class="video-text ">
                    <div class="section-head video-head">
                        <h1>Nature from the air</h1>
                        <p>Mauris consequat libero metus, nec ultricies sem efficitur quis. Integer bibendum eget metus
                            ac.Mauris consequat libero metus, nec ultricies sem efficitur quis. Integer bibendum eget
                            metus ac.Mauris consequat libero metus,
                        </p>
                        <div class="btn pd50 h-btn">
                            <a href="#">Ask Me price</a>
                        </div>
                    </div>
                </div>
            </div>
            <div class="video-top">
                <div class="video-text">
                    <div class="section-head video-head">
                        <h1>City from the air</h1>
                        <p>Mauris consequat libero metus, nec ultricies sem efficitur quis. Integer bibendum eget metus
                            ac.Mauris consequat libero metus, nec ultricies sem efficitur quis. Integer bibendum eget
                            metus ac.Mauris consequat libero metus,
                        </p>
                        <div class="btn pd50 h-btn">
                            <a href="#">Ask Me price</a>
                        </div>
                    </div>
                </div>
                <div class="video-section ">
                    <img src="img/img-left.png" alt="">
                </div>
            </div>
        </div>
    </section>
    <!-- END dorenvideo -->
    <!-- newsletter -->
    <section class="newsletter">
        <div class="container">
            <div class="newsletter-section">
                <h1>Sign up for our newsletter</h1>
                <div class="newsletter-Send">
                    <input type="email" name="" id="" placeholder="Enter Your Email">
                    <a href="#">Send</a>
                </div>
            </div>
        </div>
    </section>
    <!-- Map -->
    <section class="map">
        <iframe
            src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d14604.927911881727!2d90.3654215!3d23.7747523!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!4m3!3e6!4m0!4m0!5e0!3m2!1sen!2sbd!4v1585247433335!5m2!1sen!2sbd"
            frameborder="0" " allowfullscreen="" aria-hidden=" false" tabindex="0"></iframe>
    </section>
    <!-- End map -->
    <footer>
        <div class="container pd100">
            <p>Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. <br>
                Try awsome tool for desgners <strong> symu.co </strong></p>
        </div>
    </footer>
</body>

</html>