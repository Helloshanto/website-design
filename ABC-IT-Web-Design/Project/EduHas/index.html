<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>EduHas</title>
    <link rel="icon" href="img/our-blog-01.png" sizes="16x16" type="image/png">
    <link rel="stylesheet" href="style.css">
</head>

<body>

    <!-- Start EduHas Coding -->
    <header>
        <div class="header-top">
            <div class="container">
                <div class="col-6">
                    <div class="header-top-left">
                        <p class="fl"><i class="fas fa-phone"></i>01953332018</p>
                        <p class="fl"><i class="fas fa-envelope"></i>helloshanto@outlook.com</p>
                    </div>
                </div>
                <div class="col-6">
                    <div class="header-top-right">
                        <div class="top-social fl">
                            <a href="#" target="_blank"><i class="fab fa-facebook-square"></i></a>
                            <a href="#" target="_blank"><i class="fab fa-github"></i></a>
                            <a href="#" target="_blank"><i class="fab fa-dribbble"></i></a>
                            <a href="#" target="_blank"><i class="fab fa-behance"></i></a>
                            <a href="#" target="_blank"><i class="fab fa-stack-overflow"></i></a>
                        </div>
                        <div class="sign-in">
                            <p>login</p>
                            <p>Register</p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </header>
    <section>
        <div class="container">
            <div class="nav-section">
                <div class="logo fl"><img src="img/logo.png" alt="logo"></div>
                <div class="navbar">
                    <ul>
                        <li><a href="#">Home</a></li>
                        <li><a href="#">About</a></li>
                        <li><a href="#">Gallery</a></li>
                        <li><a href="#">Courses</a></li>
                        <li><a href="#">Blog</a></li>
                        <li><a href="#">pages</a></li>
                        <li><a href="#">contact</a></li>
                        <div class="navicon">
                            <li><a href="#"><i class="fas fa-cart-plus"></i></a></li>
                            <li><a href="#"><i class="fas fa-search"></i></a></li>
                        </div>
                    </ul>
                </div>
            </div>
        </div>
    </section>
    <section>
        <div class="slider">
            <div class="container">
                <div class="col-7">
                    <div class="slider-text-area">
                        <h1><span>Modern Education </span></h1>
                        <h1 class="mb30"> For Everyone </h1>
                        <a href="#" class="button-nav">About Us</a>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <div class="catagori-section pd100">
        <div class="container">
            <div class="section-title">
                <h3>WELCOME TO EDUCATION</h3>
                <h1>OUR COURSE CATEGORIES</h1>
                <p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a pi classical
                    Latin literature from 45 BC, making it over 2000 years old. Richard McClintock,</p>
            </div>
            <div class="ctg-course mt50">
                <div class="course-list green">
                    <img src="img/icon-01.png" alt="">
                    <h4>Photoshop</h4>
                </div>
                <div class="course-list yellow">
                    <img src="img/icon-02.png" alt="">
                    <h4>Photoshop</h4>
                </div>
                <div class="course-list blue">
                    <img src="img/icon-03.png" alt="">
                    <h4>Photoshop</h4>
                </div>
                <div class="course-list pink">
                    <img src="img/icon-04.png" alt="">
                    <h4>Photoshop</h4>
                </div>
                <div class="course-list sky">
                    <img src="img/icon-05.png" alt="">
                    <h4>Photoshop</h4>
                </div>
                <div class="course-list lemon">
                    <img src="img/icon-06.png" alt="">
                    <h4>Photoshop</h4>
                </div>
                <div class="allcourse">
                    <a href="#">All course</a>
                </div>
            </div>
        </div>
    </div>
    <section class="wechoose pd50">
        <div class="container">
            <div class="section-title pd50">
                <h3>WHY CHOOSE US</h3>
                <h1 class="white">WHY SHOULD EDUCATION </h1>
                <img src="img/title-01.png" alt="">
            </div>
            <div class="choose-part">
                <div class="col-4">
                    <div class="choose-list">
                        <img src="img/choos-01.png" alt="">
                        <h4>Apply Online</h4>
                        <p>Contrary to popular belief, Lorem Ipsum isnotimply random text. It has roots in a pi
                            classiLatin litture from 45 BC, </p>
                    </div>
                </div>
                <div class="col-4">
                    <div class="choose-list">
                        <img src="img/choos-02.png" alt="">
                        <h4>Superfast Support</h4>
                        <p>Contrary to popular belief, Lorem Ipsum isnotimply random text. It has roots in a pi
                            classiLatin litture from 45 BC, </p>
                    </div>
                </div>
                <div class="col-4">
                    <div class="choose-list">
                        <img src="img/choos-03.png" alt="">
                        <h4>Certification</h4>
                        <p>Contrary to popular belief, Lorem Ipsum isnotimply random text. It has roots in a pi
                            classiLatin litture from 45 BC, </p>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <section class="courses-list pd100" id="courses">
        <div class="container">
            <div class="section-title">
                <h3>WHY CHOOSE US</h3>
                <h1>MOST POPULAR COURSES </h1>
                <img src="img/title-02.png" alt="">
            </div>
            <div class="courses-section mt50">
                <div class="col-4">
                    <div class="same-part">
                        <img src="img/courses-02.jpg" alt="">
                        <h4>Department of Science.</h4>
                        <h6><a href="#">Political Science</a> | Duration : 4 Yr </h6>
                        <p>Contrary to popular belief, Lorem Ipsum isnotimply random text..</p>
                        <a href="#" class="course-btn">apply now</a>
                    </div>
                </div>
                <div class="col-4">
                    <div class="same-part">
                        <img src="img/courses-01.jpg" alt="">
                        <h4>Department of Science.</h4>
                        <h6><a href="#">Political Science</a> | Duration : 4 Yr </h6>
                        <p>Contrary to popular belief, Lorem Ipsum isnotimply random text..</p>
                        <a href="#" class="course-btn">apply now</a>
                    </div>
                </div>
                <div class="col-4">
                    <div class="same-part">
                        <img src="img/courses-03.jpg" alt="">
                        <h4>Department of Science.</h4>
                        <h6><a href="#">Political Science</a> | Duration : 4 Yr </h6>
                        <p>Contrary to popular belief, Lorem Ipsum isnotimply random text..</p>
                        <a href="#" class="course-btn">apply now</a>
                    </div>
                </div>
                <div class="col-4">
                    <div class="same-part">
                        <img src="img/courses-04.jpg" alt="">
                        <h4>Department of Science.</h4>
                        <h6><a href="#">Political Science</a> | Duration : 4 Yr </h6>
                        <p>Contrary to popular belief, Lorem Ipsum isnotimply random text..</p>
                        <a href="#" class="course-btn">apply now</a>
                    </div>
                </div>
                <div class="col-4">
                    <div class="same-part">
                        <img src="img/courses-05.jpg" alt="">
                        <h4>Department of Science.</h4>
                        <h6><a href="#">Political Science</a> | Duration : 4 Yr </h6>
                        <p>Contrary to popular belief, Lorem Ipsum isnotimply random text..</p>
                        <a href="#" class="course-btn">apply now</a>
                    </div>
                </div>
                <div class="col-4">
                    <div class="same-part">
                        <img src="img/courses-06.jpg" alt="">
                        <h4>Department of Science.</h4>
                        <h6><a href="#">Political Science</a> | Duration : 4 Yr </h6>
                        <p>Contrary to popular belief, Lorem Ipsum isnotimply random text..</p>
                        <a href="#" class="course-btn">apply now</a>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <section class="invitation pd100">
        <div class="container">
            <div class="section-title pb50">
                <h3>HOW TO GET YOUR DREAM WORK?</h3>
                <h1 class="white">FREE INTRODUCTORY SEMINAR</h1>
                <p class="white">Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a pi
                    classical
                    Latin literature from 45 BC, making it over 2000 years old. Richard McClintock,</p>
            </div>
            <div class="invite">
                <a href="#">RECEIVE AN INVITATION</a>
            </div>
        </div>
    </section>
    <section id="blog" class="blog pd100">
        <div class="container">
            <div class="section-title pb50">
                <h3>WELCOME TO Blog</h3>
                <h1>NEWS ABOUT EDUCATION</h1>
                <img src="img/title-02.png" alt="">
            </div>
            <div class="blog-section">
                <div class="col-4 mb50">
                    <div class="same-part">
                        <img src="img/courses-06.jpg" alt="">
                        <h4>Department of Science.</h4>
                        <h6>by : <a href="#">Shanto </a> | Date: 15/3/2020 </h6>
                        <p>Contrary to popular belief, Lorem Ipsum isnotimply random text..</p>
                        <a href="#" class="course-btn">Read More</a>
                    </div>
                </div>
                <div class="col-4 mb50">
                    <div class="same-part">
                        <img src="img/courses-06.jpg" alt="">
                        <h4>Department of Science.</h4>
                        <h6>by : <a href="#">Shanto </a> | Date: 15/3/2020 </h6>
                        <p>Contrary to popular belief, Lorem Ipsum isnotimply random text..</p>
                        <a href="#" class="course-btn">Read More</a>
                    </div>
                </div>
                <div class="col-4 mb50">
                    <div class="same-part">
                        <img src="img/courses-06.jpg" alt="">
                        <h4>Department of Science.</h4>
                        <h6>by : <a href="#">Shanto </a> | Date: 15/3/2020 </h6>
                        <p>Contrary to popular belief, Lorem Ipsum isnotimply random text..</p>
                        <a href="#" class="course-btn">Read More</a>
                    </div>
                </div>
                <div class="allcourse">
                    <a href="#">All Blog Post</a>
                </div>
            </div>
        </div>
    </section>
    <section class="about-us">
        <div class="container">
            <div class="section-title pd50">
                <h3>ABOUT US</h3>
                <h1>SOME IMPORTANT FACTS</h1>
                <img src="img/title-01.png" alt="">
            </div>
            <div class="about-section">
                <div class="col-3">
                    <div class="count">
                        <h1>50</h1>
                        <p>Teacher</p>
                    </div>
                </div>
                <div class="col-3">
                    <div class="count">
                        <h1>60</h1>
                        <p>Teacher</p>
                    </div>
                </div>
                <div class="col-3">
                    <div class="count">
                        <h1>80</h1>
                        <p>Teacher</p>
                    </div>
                </div>
                <div class="col-3">
                    <div class="count">
                        <h1>90</h1>
                        <p>Teacher</p>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <div class="textimonial pd100">
        <div class="container">
            <div class="section-title pb50">
                <h3>TESTIMONIALS</h3>
                <h1>Our Students Say</h1>
                <img src="img/title-02.png" alt="">
            </div>
            <div class="testimonial-part">
                <img src="img/testimonial-author-1.png" alt="">
                <div class="tinfo">
                    <h3>Steven Walsh</h3>
                    <p>Web Designer</p>
                </div>
                <p> If you are going to use a passage of Lorem Ipsum, you need to be
                    sure there isn't anything embarrassing hidden in the middle of text. All Lorem
                    Ipsum gators on the Internet tend to repeat</p>
            </div>
        </div>
    </div>
    <section>
        <div class="footer-top">
            <div class="container">
                <div class="col-3 f-line ">
                    <div class="f-about">
                        <div class="f-title"><img src="img/logo-2.png" alt=""></div>
                        <p>If you are going to use a passage LorIsum, you anythirassing hidden in the middle of text.
                            Iators on the Internet ten.</p>
                        <div class="f-social">
                            <a href="#" target="_blank"><i class="fab fa-facebook-square"></i></a>
                            <a href="#" target="_blank"><i class="fab fa-github"></i></a>
                            <a href="#" target="_blank"><i class="fab fa-dribbble"></i></a>
                            <a href="#" target="_blank"><i class="fab fa-behance"></i></a>
                            <a href="#" target="_blank"><i class="fab fa-stack-overflow"></i></a>
                        </div>
                    </div>
                </div>
                <div class="col-3 f-line ">
                    <div class="qlink">
                        <div class="f-title">Qucik Link</div>
                        <a href="#">Privacy Statement</a>
                        <a href="#">Privacy Statemen</a>
                        <a href="#">Privacy Stateme</a>
                        <a href="#">Privacy Statem</a>
                        <a href="#">Privacy State</a>
                        <a href="#">Privacy Stat</a>
                        <a href="#">Privacy Sta</a>
                    </div>
                </div>
                <div class="col-3 f-line ">
                    <div class="f-corse">
                    <div class="f-title">POPULAR COURSES</div>
                        <img src="img/blog-01.jpg" alt="" class="fl">
                        <a href="#">Department of Science.</a>
                        <p>Duration : 4 Yr</p>
                    </div>
                    <div class="f-corse">
                        <img src="img/blog-02.jpg" alt="" class="fl">
                        <a href="#">Department of Science.</a>
                        <p>Duration : 4 Yr</p>
                    </div>
                </div>
                <div class="col-3">
                    <div class="fsend">
                        <div class="f-title">NEWSLETTER</div>
                        <p>Enter your email and we'll send you more information of.</p>
                        <input type="email" placeholder="Your Email Address">
                        <i class="far fa-paper-plane"></i>
                        <br><br><br>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <footer>
        <div class="container">
            <p>Copyright© 2018 EduHas,Designed by HasTech | <span class="tomato"> Redesign for practice -shanto</span></p>
        </div>
    </footer>
    <!-- Font awesome -->
    <script src="https://kit.fontawesome.com/40aed592e2.js" crossorigin="anonymous"></script>
    
</body>

</html>