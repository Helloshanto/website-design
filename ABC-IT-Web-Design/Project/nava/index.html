<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>nava</title>

    <!-- font asesome -->
    <link rel="stylesheet" href="assets/font-awesome/all.css">
    <link rel="stylesheet" href="assets/font-awesome/all.min.css">

    <!-- main stylesheet -->
    <link rel="stylesheet" href="style.css">

</head>

<body>
    <!-- start coding -->

    <!-- Start Header -->
    <header>
        <div class="container">
            <div class="nav-section">
                <div class="logo fl">
                    <img src="assets/img/logo.png" alt="">
                </div>
                <div class="nav fr">
                    <ul>
                        <li><a href="#">TOP DESTINATIONS</a></li>
                        <li><a href="#">ADD YOUR BOAT</a></li>
                        <li>
                            <div class="signup">
                                <a href="#">Sign-up</a>
                            </div>
                        </li>
                    </ul>
                </div>
            </div>
            <div class="slider-text">
                <h1>RESERVE A BOAT NOW!</h1>
                <h4>and set your sails into paradise!</h4>
                <div class="slider-search">
                    <input type="text" placeholder="What is your DESTINATION, SAILOR?">
                    <a href="#"><i class="fas fa-anchor"></i> Find A Boat</a>
                </div>
            </div>
        </div>
    </header>
    <!-- End Start Header -->
    <!-- boat-info -->
    <section>
        <div class="container">
            <div class="boat-info">
                <div class="col-25">
                    <div class="boat-info-section boat-br">
                        <h3>2,300</h3>
                        <p>Destinations</p>
                    </div>
                </div>
                <div class="col-25">
                    <div class="boat-info-section boat-br">
                        <h3>2,300</h3>
                        <p>Destinations</p>
                    </div>
                </div>
                <div class="col-25">
                    <div class="boat-info-section boat-br">
                        <h3>2,300</h3>
                        <p>Destinations</p>
                    </div>
                </div>
                <div class="col-25">
                    <div class="boat-info-section">
                        <h3>2,300</h3>
                        <p>Destinations</p>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <!-- End boat-info -->
    <!-- EXPLORE section -->
    <section>
        <div class="container pd100">
            <div class="setion-title pd50">
                <h3>EXPLORE OUR TOP DESTINATIONS</h3>
                <p>Lorem ipsum dolor sit amet, consectetur adipisticing elit, sed do eiusmod tempor incididunt ut labore
                    et .</p>
            </div>
            <div class="EXPLORE">
                <div class="col-70">
                    <div class="exp-section">
                        <img src="assets/img/location-bg-1.png" alt="">
                        <div class="exp-top">
                            <a href="#"><i class="fas fa-anchor"></i> 568</a>
                        </div>
                        <div class="exp-bottom">
                            <h3>Ibiza</h3>
                            <p>Spain</p>
                        </div>
                    </div>
                </div>
                <div class="col-30">
                    <div class="exp-section">
                        <img src="assets/img/location-bg-2.png" alt="">
                        <div class="exp-top">
                            <a href="#"><i class="fas fa-anchor"></i> 215</a>
                        </div>
                        <div class="exp-bottom">
                            <h3>Fguerira da Foz</h3>
                            <p>portugal</p>
                        </div>
                    </div>
                </div>
                <div class="col-30">
                    <div class="exp-section">
                        <img src="assets/img/location-bg-3.png" alt="">
                        <div class="exp-top">
                            <a href="#"><i class="fas fa-anchor"></i> 657</a>
                        </div>
                        <div class="exp-bottom">
                            <h3>Palma de Mallorca</h3>
                            <p>Spain</p>
                        </div>
                    </div>
                </div>
                <div class="col-30">
                    <div class="exp-section">
                        <img src="assets/img/location-bg-4.png" alt="">
                        <div class="exp-top">
                            <a href="#"><i class="fas fa-anchor"></i> 345</a>
                        </div>
                        <div class="exp-bottom">
                            <h3>portofino</h3>
                            <p>italy</p>
                        </div>
                    </div>
                </div>
                <div class="col-30">
                    <div class="exp-section">
                        <img src="assets/img/location-bg-2.png" alt="">
                        <div class="exp-top">
                            <a href="#"><i class="fas fa-anchor"></i> 325</a>
                        </div>
                        <div class="exp-bottom">
                            <h3>Fguerira da Foz</h3>
                            <p>portugal</p>
                        </div>
                    </div>
                </div>
                <div class="exp-button pd50">
                    <a href="#">EXPLORE MORE DESTINATIONS</a>
                </div>
            </div>
        </div>
    </section>
    <!--END EXPLORE section -->
    <!-- OWN BOAT? -->
    <section class="ownboat pd100">
        <div class="container">
            <div class="ownboat-section">
                <h3>HAVE YOUR OWN BOAT?</h3>
                <p>navigare necesse est, vivere non est necesse</p>
                <a href="#">Make money with your boat</a>
            </div>
        </div>
    </section>

    <!-- END OWN BOAT? -->
    <!-- FEATURED BOATS -->
    <section class="pd100">
        <div class="container">
            <div class="setion-title pd50">
                <h3>FEATURED BOATS</h3>
                <p>Lorem ipsum dolor sit amet, consectetur adipisticing elit, sed do eiusmod tempor incididunt ut labore
                    et .</p>
            </div>
            <div class="EXPLORE">
                <div class="col-50">
                    <div class="exp-section">
                        <img src="assets/img/boat-01.png" alt="">
                        <div class="exp-top fe-top">
                            <a href="#"><i class="fas fa-anchor"></i>€220 / day </a>
                        </div>
                        <div class="exp-bottom fe-bottom">
                            <h3>Ibiza</h3>
                            <p>Spain</p>
                        </div>
                    </div>
                </div>
                <div class="col-50">
                    <div class="exp-section">
                        <img src="assets/img/boat-02.png" alt="">
                        <div class="exp-top fe-top">
                            <a href="#"><i class="fas fa-anchor"></i>€820 / day </a>
                        </div>
                        <div class="exp-bottom fe-bottom">
                            <h3>Fguerira da Foz</h3>
                            <p>portugal</p>
                        </div>
                    </div>
                </div>
                <div class="col-50">
                    <div class="exp-section">
                        <img src="assets/img/boat-03.png" alt="">
                        <div class="exp-top fe-top">
                            <a href="#"><i class="fas fa-anchor"></i>€226 / day</a>
                        </div>
                        <div class="exp-bottom fe-bottom">
                            <h3>Palma de Mallorca</h3>
                            <p>Spain</p>
                        </div>
                    </div>
                </div>
                <div class="col-50">
                    <div class="exp-section">
                        <img src="assets/img/boat-04.png" alt="">
                        <div class="exp-top fe-top">
                            <a href="#"><i class="fas fa-anchor"></i>€320 / day</a>
                        </div>
                        <div class="exp-bottom fe-bottom">
                            <h3>portofino</h3>
                            <p>italy</p>
                        </div>
                    </div>
                </div>
                <div class="exp-button pd50">
                    <a href="#">LOAD MORE BOATS</a>
                </div>
            </div>
        </div>
    </section>
    <!-- End FEATURED BOATS -->
    <!-- footer -->
    <footer class="pd50">
        <div class="footer">
            <div class="container">
                <div class="col-20">
                    <h6>TOP LOCATIONS</h6>
                    <div class="footer-menu">
                        <ul>
                            <li><a href="#">Lorem ipsum dolor</a></li>
                            <li><a href="#">Sit amet consectetur</a></li>
                            <li><a href="#">Adipisicing elit</a></li>
                            <li><a href="#">Eiusmod tempor</a></li>
                            <li><a href="#">Incididunt ut labore </a></li>
                        </ul>
                    </div>
                </div>
                <div class="col-20">
                    <h6>TOP LOCATIONS</h6>
                    <div class="footer-menu">
                        <ul>
                            <li><a href="#">Lorem ipsum dolor</a></li>
                            <li><a href="#">Sit amet consectetur</a></li>
                            <li><a href="#">Adipisicing elit</a></li>
                            <li><a href="#">Eiusmod tempor</a></li>
                        </ul>
                    </div>
                </div>
                <div class="col-40">
                    <h6>SUBSCRIBE TO OUR NEWSLETTER</h6>
                    <div class="footer-sub">
                        <input type="email" placeholder="Enter Your Email">
                        <a href="#">subscribe</a>
                        <div class="fsocial">
                            <a href="#"><i class="fab fa-facebook-f"></i></a>
                            <a href="#"><i class="fab fa-twitter"></i></a>
                            <a href="#"><i class="fab fa-google-plus-g"></i></a>
                        </div>
                    </div>
                </div>
                <div class="col-20">
                    <h6>TOP LOCATIONS</h6>
                    <div class="footer-menu">
                        <ul>
                            <li><a href="#">Lorem ipsum dolor</a></li>
                            <li><a href="#">Sit amet consectetur</a></li>
                            <li><a href="#">Adipisicing elit</a></li>
                            <li><a href="#">Eiusmod tempor</a></li>
                            <li><a href="#">Incididunt ut labore </a></li>
                        </ul>
                    </div>
                </div>
            </div>
        </div>
    </footer>
    <!-- End footer -->
    <!-- End start coding -->
</body>

</html>