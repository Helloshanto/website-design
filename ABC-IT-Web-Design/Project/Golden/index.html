<!DOCTYPE html>
<html lang="en">

    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <title>Golden-exam</title>
        <link rel="stylesheet" href="style.css">
    </head>

    <body>
        <!-- Start site coding -->
        <!-- Start Header section -->
        <header>
            <div class="container">
                <div class="nav-section">
                    <div class="logo"><img src="img/Logo.png" alt=""></div>
                    <div class="nav">
                        <ul>
                            <li><a href="#">Home</a></li>
                            <li><a href="#Service">Service</a></li>
                            <li><a href="#portfolio">portfolio</a></li>
                            <li><a href="#">About</a></li>
                            <li><a href="#">Contact</a></li>
                        </ul>
                    </div>
                </div>
            </div>
            <div class="container">
                <div class="slider">
                    <h3>Welcome To Our Studio!</h3>
                    <h1>Itâ€™s nice to Meet you</h1>
                    <a href="#">Tell Me More</a>
                </div>
            </div>
        </header>
        <!-- End Header Section -->
        <!-- Start Service section -->
        <section class="service pd100" id="Service">
            <div class="container">
                <div class="section-title">
                    <h1>Services</h1>
                    <p>Proin iaculis purus consequat sem cure. </p>
                </div>
                <div class="service-section">
                    <div class="col-4">
                        <div class="service-part">
                            <i class="fas fa-laptop-code"></i>
                            <h3>E-Commerce</h3>
                            <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean
                                rhoncus posuere odio in tincidunt.</p>
                        </div>
                    </div>
                    <div class="col-4">
                        <div class="service-part">
                            <i class="fas fa-shopping-bag"></i>
                            <h3>E-Commerce</h3>
                            <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean
                                rhoncus posuere odio in tincidunt.</p>
                        </div>
                    </div>
                    <div class="col-4">
                        <div class="service-part">
                            <i class="fas fa-unlock-alt"></i>
                            <h3>E-Commerce</h3>
                            <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean
                                rhoncus posuere odio in tincidunt.</p>
                        </div>
                    </div>

                </div>
            </div>
        </section>
        <!-- End Service section -->
        <!-- Portfolio Section -->
        <section class="portfolio pd100" id="portfolio">
            <div class="container">
                <div class="section-title">
                    <h1>Portfolio</h1>
                    <p>Proin iaculis purus consequat sem cure. </p>
                </div>
                <div class="portfolio-section">
                    <div class="col-4">
                        <div class="portfolio-part">
                            <img src="img/portfolio-01.png" alt="">
                            <h3>Ebony & Lvory</h3>
                            <p>Branding</p>
                        </div>
                    </div>
                    <div class="col-4">
                        <div class="portfolio-part">
                            <img src="img/portfolio-02.png" alt="">
                            <h3>Ebony & Lvory</h3>
                            <p>Branding</p>
                        </div>
                    </div>
                    <div class="col-4">
                        <div class="portfolio-part">
                            <img src="img/portfolio-03.png" alt="">
                            <h3>Ebony & Lvory</h3>
                            <p>Branding</p>
                        </div>
                    </div>
                </div>
            </div>
        </section>
        <!--End  Portfolio Section -->
        <!--End  site coding  -->

        <!-- Font awesome -->
        <script src="https://kit.fontawesome.com/40aed592e2.js" crossorigin="anonymous"></script>
    </body>

</html>